bagvalidate/errors.go

```diff
@@ -2,19 +2,17 @@ package bagvalidate
 
 import (
 	"errors"
-	"fmt"
 	"strings"
-
-	bagit_gython "github.com/artefactual-labs/bagit-gython"
 )
 
+var ErrNotABag = errors.New("not a bag")
+
 var ErrInvalid = errors.New("invalid")
 
-func convertError(err error) error {
-	if errors.Is(err, bagit_gython.ErrInvalid) {
-		msg, _ := strings.CutPrefix(err.Error(), "invalid: ")
-		err = fmt.Errorf("%w: %s", ErrInvalid, msg)
-	}
+func RemovePathFromError(path string, err error) string {
+	// Remove path from validation messages.
+	message := strings.Replace(err.Error(), path+" is invalid: ", "", 1)
 
-	return err
+	// Convert to lower case.
+	return strings.ToLower(message)
```

Comment:
Why convert the message to lower case?

Reply:
Ah, sorry... I was trying to adhere to https://google.github.io/styleguide/go/decisions.html#:~:text=Error%20strings%20should%20not%20be,%22Something%20bad%20happened.%22) but re-reading it it just seems to be to not capitalize the first letter. I'll remove the lower casing.